app/livechat/server/api/v1/contact.js

```diff
@@ -28,12 +28,12 @@ API.v1.addRoute(
 				return API.v1.failure(e);
 			}
 		},
-		get() {
+		async get() {
 			check(this.queryParams, {
 				contactId: String,
 			});
 
-			const contact = Promise.await(LivechatVisitors.findOneById(this.queryParams.contactId));
+			const contact = await LivechatVisitors.findOneById(this.queryParams.contactId);
```

Comment:
from what I see, `LivechatVisitors` is not a raw model, so it was not even needed to `await` here 😬

Reply:
Good catch 👀 this was not a raw model and we were "awaiting" it lol. Let me fix it

Reply:
yep.. that is not an issue though.. not sure how harmful it is to let the `await` there

Reply:
I don't think it affects 🤔 there are other places where we "promise.await" on meteor models lol. Maybe the biggest issue is the overhead on node 🤷‍♂️